benches/benchmarks.rs

```diff
@@ -74,6 +74,67 @@ pub fn benchmark(c: &mut Criterion) {
             },
         );
     }
+
+    {
+        let mut uncontended = c.benchmark_group("uncontended guarded");
+        uncontended.bench_function("read output", |b| b.iter(|| *output.output_buffer()));
+        uncontended.bench_function("clean update", |b| {
+            b.iter(|| {
+                output.update();
+            })
+        });
+        uncontended.bench_function("clean receive", |b| b.iter(|| *output.read()));
+        uncontended.bench_function("write input", |b| {
+            b.iter(|| {
+                *input.input_buffer_publisher() = black_box(0);
+            })
+        });
+        uncontended.bench_function("send", |b| {
+            b.iter(|| *input.input_buffer_publisher() = black_box(0))
+        });
+        uncontended.bench_function("transmit", |b| {
+            b.iter(|| {
+                *input.input_buffer_publisher() = black_box(0);
+                *output.read()
+            })
+        });
```

Comment:
"read output", "clean update" and "clean receive" are a copy of the existing benchmarks above, and "write input" and "send" are identical to each other. This is not nice when each new benchmark adds 5s of running time to the benchmark suite.

Instead, consider remplacing this with a few additions to the existing "uncontended" benchmark group above:

- This group's "write input"/"send" benchmark could become a new "guarded write+send" benchmark in the existing "uncontended" group.
- This group's "transmit" benchmark could become a new "guarded transmit" benchmark in the existing "uncontended group.